### PR TITLE
Address issue 70 - update codepoints

### LINKED ISSUE #70: Update codepoints for draft-irtf-cfrg-vdaf-12
We will likely re-assign codepoints in the next VDAF draft: https://github.com/cfrg/draft-irtf-cfrg-vdaf/pull/417

### PR DESCRIPTION
**Pull request by junyechen1996**
Closes #70 .

### CODE REVIEW

File: draft-wang-ppm-dap-taskprov.md [hunk 1]
```diff
@@ -278,17 +281,13 @@ differential privacy (DP). The opaque `dp_config` contains the following structu
 enum {
     reserved(0), /* Reserved for testing purposes */
     none(1),
-    aggregator_discrete_gaussian(5),
```
Comment (junyechen1996) [inline draft-wang-ppm-dap-taskprov.md:281]: Don't think this is standardized yet? @cjpatton

Comment (cjpatton) [reply draft-wang-ppm-dap-taskprov.md:281]: You're right, I'm not sure how this got added. It looks like I did somehow 8f806955

**Review by cjpatton — APPROVED** [review]
Rebasing should make CI pass.

**Comment by junyechen1996** [comment]
@cjpatton I will wait for DAP-12 to be cut.

**Pull request merged**